controllers/DefaultController.php

```diff
@@ -98,7 +98,10 @@ public function beforeAction($action)
 			if (!Yii::app()->user->isGuest) {
 				$this->redirect(array('profile'));
 				return false;
-			}
+			}           
+            if(class_exists('SubscribeBehavior')) {
+                $this->attachBehavior('subscribeUser', new SubscribeBehavior);
```

Comment:
This creates an unecessary dependency with yii-nfy and needs to be solved differently.